addons/slack/install.js

```diff
@@ -13,64 +13,94 @@ const getTemplateFiles = (fileNames) =>
   }, {});
 
 module.exports = async (ctx) => {
-  let storageSpecification = await Sdk.createStorage(ctx);
+  if (ctx.caller.permissions) {
+    // Create the Add-On Handler
 
-  // Create the Add-On Handler
-  try {
-    await Sdk.createFunction(ctx, {
-      configurationSerialized: `# Add-on configuration settings
+    // Capture resource path and remove it as we will not need it in the addon instance
+    const eventsResourcePathKey = 'fusebit_events_resource_path';
```

Comment:
Pull constants out of functions and move them to the top of the file.

Reply:
Fixed